src/AppInstallerCLIE2ETests/FeaturesCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AppInstallerCLIE2ETests
+{
+    using NUnit.Framework;
+
+    public class FeaturesCommand
+    {
+        [Test]
+        public void DisplayFeatures()

```

Comment:
A test that enables a feature would be good.

Reply:
Added test to enable experimental features and check if they're enabled on winget